router/transformer/transformer.go

```diff
@@ -371,16 +371,14 @@ func (trans *handle) doProxyRequest(ctx context.Context, proxyReqParams *ProxyRe
 			statusCode: http.StatusInternalServerError,
 			err:        err,
 		}
-	} else if resp.StatusCode == http.StatusNotFound {
-		// Actually Router wouldn't send any destination to proxy unless it already exists
-		// But if accidentally such a request is sent, we'd probably need to handle for better
-		// understanding of the response
-		notFoundErr := fmt.Errorf(`post "%s" not found`, req.URL)
-		trans.logger.Errorf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Client.Do Failure for %[1]v, with %[3]v`, destName, jobID, notFoundErr)
+	} else if resp.StatusCode != http.StatusOK {
```

Comment:
Should we handle this condition as when resp.StatusCode is not 2xx, then you throw an error ? Or should we handle for status-codes that are > 500, throw a non-success error ?

Reply:
🤔